disnake/components.py

```diff
@@ -210,18 +213,17 @@ def to_dict(self) -> ButtonComponentPayload:
         return payload
 
 
-class SelectMenu(Component):
-    """Represents a select menu from the Discord Bot UI Kit.
+class BaseSelectMenu(Component):
+    """Represents an abstract select menu from the Discord Bot UI Kit.
 
     A select menu is functionally the same as a dropdown, however
     on mobile it renders a bit differently.
 
-    .. note::
+    The currently supported select menus are:
 
-        The user constructible and usable type to create a select menu is
```

Comment:
this line should not be deleted, but modified.

Reply:
should be resolved as of https://github.com/DisnakeDev/disnake/pull/800/commits/46c30453d0d22f6c4c4a2ce5eb481dcea9ba9941, if I understood correctly